extensions/linear/src/api/getMilestones.ts

```diff
@@ -0,0 +1,53 @@
+import { ProjectMilestone } from "@linear/sdk";
+import { getLinearClient } from "../helpers/withLinearClient";
+
+export type MilestoneResult = Pick<
+  ProjectMilestone,
+  "id" | "name" | "description" | "targetDate" | "archivedAt" | "createdAt" | "updatedAt"
+> & {
+  // Linear doesn't seem to expose the startDate property even though we can retrieve it
+  startDate?: string;
+} & {
+  project: { nodes: { id: string; key: string }[] };
+};
+
+const milestoneFragment = `
+  id
+  name
+  description
+  targetDate
+  archivedAt
+  createdAt
+  project {
+      id
+  }
+  sortOrder
+  updatedAt
+`;
```

Comment:
Are all milestones properties useful here? As far as it looks, `archivedAt`, `createdAt`, and `description` may be useless.

Reply:
That's a good point! I will remove it for now.